src/lexer.rs

```diff
@@ -494,7 +678,7 @@ pub fn lex(input: &str, allow_trailing_operators: bool, default_degree: Unit) ->
           },
         }
       },
-      // decide if " is inch of inch of mercury
+      // decide if " is inch or inch of mercury
```

Comment:
I think `inch` and `inch of mercury` should be quoted, to more clearly delineate what is being matched.

Reply:
Like \`"hg`?

Reply:
Nah, I just mean to make this comment easier to read at a glance:

```
// decide if " is 'inch' or 'inch of mercury'
```

Reply:
Ah right, will do that. Was reading this inside VSCode which showed it at the wrong like.